.travis.yml

```diff
@@ -29,7 +29,8 @@ install:
 #    - git clone --depth 1 git://github.com/astropy/ci-helpers.git
     - git clone --depth 1 -b all-the-fixes git://github.com/djhoese/ci-helpers.git    
     - source ci-helpers/travis/setup_conda.sh
-script: coverage run --source=pyspectral setup.py test
+script:
+  - pytest --cov=./
```

Comment:
you might want to do `pip install .` first, otherwise the coverage might not work